### PR TITLE
Add backup and restore content for tiup-cluster.md

### PR DESCRIPTION
**Pull request by qqqdan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by nexustar — APPROVED** [review]
LGTM

File: tiup/tiup-cluster.md [hunk 1]
```diff
@@ -693,3 +693,20 @@ TiUP 相关的数据都存储在用户 home 目录的 `.tiup` 目录下，若要
 > **注意：**
 >
 > 为了避免中控机磁盘损坏等异常情况导致 TiUP 数据丢失，建议定时备份 `.tiup` 目录。
+
+## 备份与恢复集群部署和运维所需的 meta 文件
+
+如果运维所需的 meta 文件丢失，会导致无法继续使用 tiup 管理集群，建议通过以下方式定期备份 meta 文件。
```
Comment (shichun-0415) [inline tiup/tiup-cluster.md:699]:
```suggestion
如果运维所需的 meta 文件丢失，会导致无法继续使用 TiUP 管理集群，建议通过以下方式定期备份 meta 文件。
```

File: tiup/tiup-cluster.md [hunk 2]
```diff
@@ -693,3 +693,20 @@ TiUP 相关的数据都存储在用户 home 目录的 `.tiup` 目录下，若要
...
+
+```bash
+tiup cluster meta backup ${cluster_name}
+
+```
```
Comment (shichun-0415) [inline tiup/tiup-cluster.md:704]:
```suggestion
```

```

File: tiup/tiup-cluster.md [hunk 3]
```diff
@@ -693,3 +693,20 @@ TiUP 相关的数据都存储在用户 home 目录的 `.tiup` 目录下，若要
...
+可以使用以下方法恢复 meta 文件：
```
Comment (shichun-0415) [inline tiup/tiup-cluster.md:705]: 什么时候需要恢复 meta 文件呢

File: tiup/tiup-cluster.md [hunk 4]
```diff
@@ -693,3 +693,20 @@ TiUP 相关的数据都存储在用户 home 目录的 `.tiup` 目录下，若要
...
+
+```bash
+tiup cluster meta restore ${cluster_name} ${backup_file}
+```
+> **注意：**
```
Comment (shichun-0415) [inline tiup/tiup-cluster.md:710]:
```suggestion

> **注意：**
```

File: tiup/tiup-cluster.md (hunk 3 above)
Comment (qqqdan) [reply tiup/tiup-cluster.md:703]: backup 和 restore 是一对的，恢复的场景各种情况都有，感觉不用描述读者都可以理解哈

File: tiup/tiup-cluster.md [hunk 5]
```diff
@@ -693,3 +693,19 @@ TiUP 相关的数据都存储在用户 home 目录的 `.tiup` 目录下，若要
...
 >
 > 为了避免中控机磁盘损坏等异常情况导致 TiUP 数据丢失，建议定时备份 `.tiup` 目录。
+
+## 备份与恢复集群部署和运维所需的 meta 文件
+
+如果运维所需的 meta 文件丢失，会导致无法继续使用 TiUP 管理集群，建议通过以下方式定期备份 meta 文件。
+
+```bash
+tiup cluster meta backup ${cluster_name}
+可以使用以下方法恢复 meta 文件：
```
Comment (shichun-0415) [inline tiup/tiup-cluster.md:703]:
```suggestion

可以使用以下方法恢复 meta 文件：
```

File: tiup/tiup-cluster.md [hunk 6]
```diff
@@ -693,3 +693,19 @@ TiUP 相关的数据都存储在用户 home 目录的 `.tiup` 目录下，若要
...
+
+```bash
+tiup cluster meta restore ${cluster_name} ${backup_file}
+```
+
+> **注意：**
+>
+> 恢复操作会覆盖当前的 meta 文件，建议在 meta 文件丢失的情况下才进行恢复。
```
Comment (shichun-0415) [inline tiup/tiup-cluster.md:711]:
```suggestion
> 恢复操作会覆盖当前的 meta 文件，建议仅在 meta 文件丢失的情况下进行恢复。
```

File: tiup/tiup-cluster.md [hunk 7]
```diff
@@ -693,3 +693,20 @@ TiUP 相关的数据都存储在用户 home 目录的 `.tiup` 目录下，若要
 > **注意：**
 >
 > 为了避免中控机磁盘损坏等异常情况导致 TiUP 数据丢失，建议定时备份 `.tiup` 目录。
+
+## 备份与恢复集群部署和运维所需的 meta 文件
+
+如果运维所需的 meta 文件丢失，会导致无法继续使用 TiUP 管理集群，建议通过以下方式定期备份 meta 文件。
+
+```bash
+tiup cluster meta backup ${cluster_name}
```
Comment (shichun-0415) [inline tiup/tiup-cluster.md:702]:
```suggestion
```bash
tiup cluster meta backup ${cluster_name}
```
```

**Review by shichun-0415 — APPROVED** [review]

**Pull request merged**